docs/docs/answering-it-security.md

```diff
@@ -7,7 +7,7 @@ At larger companies, such as the Fortune 500, there are Security teams that audi
 If security engineers are interested in your project, some talking points that can help answer their questions include:
 
 - Because Gatsby compiles your site to flat files, rather than having running app servers and databases users are targeting, it reduces the attack surface of the site to outsiders.
-- Gatsby adds a layer of indirection which obscures your CMS -- so even if your CMS _is_ vulnerable, bad actors have no idea where to find it. This is in contrast to systems where bad actors can easily locate the admin dashboard at, e.g., `/wp-admin` and attempt to hack in.
+- Gatsby adds a layer of indirection which obscures your CMS -- so even if your CMS _is_ vulnerable, bad actors have no idea where to find it. This is in contrast to systems where bad actors can be locate the admin dashboard at, e.g., `/wp-admin` and attempt to hack in.
```

Comment:
I think this is not an example of the issue outlined in the linked issue.

Additionally, I'm certain the change is incorrect.